docs/documentation/mission_module/mission_file/segments.rst

```diff
@@ -29,7 +31,7 @@ Python implementation.
 All parameters of the Python constructor can be set in the mission file (except for
 :code:`propulsion` and :code:`reference_area` that are set within the mission module).
 Most of these parameters are scalars and can be set as described :ref:`here<setting-values>`.
-The segment target is a special parameter, detailed in :ref:`further section<segment-target>`.
+The segment target is a special parameter, detailed in :ref:`further section<segment-parameter-target>`.
```

Comment:
so is it a parameter or a segment? For what I understand it cannot be called on his own but only as a parameter of another segment no?